EXAMPLES.md

```diff
@@ -277,9 +277,16 @@ additionalArguments:
 - "--experimental.plugins.bouncer.version=v1.1.9"
 ```
 
-# Use Traefik Let's Encrypt Integration with CloudFlare
+# Use Traefik native Let's Encrypt Integration, without cert-manager
+
+This example is with CloudFlare, see here for other providers.
+
+In Traefik Proxy, ACME certificates are stored in a JSON file that needs to have a
```

Comment:
I would change that to:
In Traefik Proxy, ACME certificates are stored in a JSON file.
This file needs to have `0600` permissions, meaning, only the owner of the file has full read and write access to it.

By default, Kubernetes recursively changes ownership and
permissions for the content of each volume.
An initContainer is used to avoid an issue on this sensitive file.

Reply:
Is this better now ?

Reply:
Yes, perfect!